### ISSUE TITLE
navigation using screen reader on mobile doesn't work

### ISSUE THREAD

**Issue by cptvitamin** [gnab/remark#118]
There is a gesture conflict on mobile for users of screen readers like Voiceover iOS and TalkBack on Android where swipe is equivalent to "tabbing" from focusable element to focusable element.  

Possible solutions: create links to next and previous following the slide numbers (can make them visually hidden offscreen so they don't appear on screen for average users).
If you would like a solution that also fixes this issue for anyone on a device that doesn't support swipe, you could have large transparent buttons on the left and right edge of the screen that appear on focus/hover that navigate to next and prev slide.
